hack/tools/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/metal3-io/hardware-classification-controller
+
+go 1.16
+
+require (
+	github.com/go-logr/logr v0.2.1
+	github.com/metal3-io/baremetal-operator v0.0.0-20201006073612-56a49dc7016a
+	github.com/onsi/ginkgo v1.12.1
+	github.com/onsi/gomega v1.10.1
+	github.com/pkg/errors v0.9.1
+	github.com/stretchr/testify v1.6.1
+	k8s.io/apiextensions-apiserver v0.18.6 // indirect
+	k8s.io/apimachinery v0.19.0
+	k8s.io/client-go v0.19.0
+	k8s.io/klog v1.0.0
+	sigs.k8s.io/cluster-api v0.3.6
```

Comment:
This seems to be quiet old release of CAPI, is that on purpose?

Reply:
Yes, changes are big. Will raise separate PR for this.